src/helics/shared_api_library/MessageFederate.h

```diff
@@ -517,7 +517,7 @@ HELICS_EXPORT int helicsMessageGetRawDataSize(helics_message_object message);
  * @endPythonOnly
  */
 HELICS_EXPORT void
-    helicsMessageGetRawData(helics_message_object message, void* data, int maxMessagelen, int* actualSize, helics_error* err);
+    helicsMessageGetRawData(helics_message_object message, void* data, int maxMessageLength, int* actualSize, helics_error* err);
```

Comment:
I am fine with the name change, but this also needs to be modified in all the swig files since they key of the variable name for this function.

Reply:
Resolved in 7e460d0.